### PR TITLE
Add the bot whose name shall not be mentioned in this title

### PR DESCRIPTION
**Pull request by bkeepers**
The WIP bot is awesome and very simple to use. It is now installed on all repositories in the @probot organization, and want to add it to the website.

- [x] Get @gr2m's blessing
- [x] Get a screenshot of the bot in action
- [x] Update the description to with a pitch on what it does

### CODE REVIEW

**Comment by gr2m** [comment]
very meta, this PR :D you definitely have my blessing.

Can the screenshot also be a gif? Maybe that’d explain it the best?

The avatar is just a steal from https://robohash.org/, if anyone has ideas from something better, let me know :)

**Comment by bkeepers** [comment]
> Can the screenshot also be a gif? Maybe that’d explain it the best?

👍 Do you want to create it, or want me to?

**Comment by gr2m** [comment]
I would appreciate any help right now @bkeepers <3 No worries if you don’t get to it, I will eventually :) If you get to it, I’d love a PR to the wip-bot README, too :)

File: _apps/wip-bot.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+title: Work In Progress
+description: Prevent merging of Pull Requests with "WIP" in the title
+slug: wip-bot
```
Comment (gr2m) [inline _apps/wip-bot.md]: it’s now just `wip`, thanks so much @minamimonji 💐

**Comment by gr2m** [comment]
we haz gif: https://github.com/gr2m/wip-bot

**Comment by gr2m** [comment]
@bkeepers I’ve added the URL to the gif and a short description – anything else?

**Pull request merged**